### DISCUSSION TITLE
Execution default-cli of goal org.jreleaser:jreleaser-maven-plugin:1.10.0:full-release failed: Unexpected error

### DISCUSSION THREAD

**Discussion by heshanthenura** [jreleaser/jreleaser#1619] [Q&A]
Why I'm getting this error?

![image](https://github.com/jreleaser/jreleaser/assets/75155192/4a83fabf-8bc1-4cab-b733-e6567dea8b3a)
[trace.log](https://github.com/jreleaser/jreleaser/files/14457495/trace.log)

**Comment by aalmiray** [comment]
The artifacts you deployed triggered validation constraints on the server. Log into the Sonatype UI, search for the staged repository and look for the activities that caused errors. 

Alternatively, upgrade to `v1.11.0`, try again and inspect the `target/jreleaser/trace.log` file. 

One more thing, it'd be better if you paste text instead of attaching screenshots of errors 😉

**Reply by heshanthenura** [reply]
Those are the errors

Btw what is that they ask us to upload?

![image](https://github.com/jreleaser/jreleaser/assets/75155192/e9561448-1c48-4d7c-85f1-b879ffcb86b5)
![image](https://github.com/jreleaser/jreleaser/assets/75155192/e2d3bd26-8a01-4038-a730-24ad6ec2b282)
![image](https://github.com/jreleaser/jreleaser/assets/75155192/b1b073a0-8433-4cf7-b324-43d2028517bb)

**Reply by heshanthenura** [reply]
It worked finally
![image](https://github.com/jreleaser/jreleaser/assets/75155192/3d52a539-d423-4686-ba99-b2cb8fffbf3f)

**Reply by heshanthenura** [reply]
Now where can I find my library?

**Reply by aalmiray** [reply]
It should be available from Maven Central just like any other dependency you consume from Maven or Gradle.

**Reply by heshanthenura** [reply]
Thank you all went well. Thanks for helping 🫡🫡🫡🫡🫡
